src/main/resources/psp_config_file.json

```diff
@@ -3533,8 +3533,8 @@
     "ABI08365": {
         "logo": "https://assets.cdn.io.italia.it/logos/abi/08365.png",
         "fiscalCode": "00945950780",
-        "name": "BCC dell'Alto Tirreno della Calabria Verbicaro",
-        "companyName": "Banca Di Credito Cooperativo Dell'Alto Tirreno Della Calabria Verbicaro (Provincia Di Cosenza) Societ\u00e0 Cooperativa",
+        "name": "BCC Calabria Nord",
```

Comment:
this [information](https://github.com/pagopa/pagopa-template-receipt-pdf/blob/main/py-scripts/README.md) are deprecated?

Reply:
Yes, they are deprecated because the data source is no longer the contract CRM sheet, so we need to change the config generation script in order to get data from new source (maybe SAP)